server/src/concepts/height/useCases/addNewHeight.js

```diff
@@ -0,0 +1,16 @@
+import createHeight from "../repositories/commands.js";
+
+const addNewHeight = async (data) => {
```

Comment:
We should also check if the same height already exists. I saw that you have function for this already.